src/content/app/genome-browser/components/drawer/drawer-views/gene-summary/GeneSummary.scss

```diff
@@ -1,8 +1,23 @@
 @import 'src/styles/common';
 
+$drawerWindowWidth: 45px;
+$sidebarWithToolstripWidth: calc(320px + 46px);
+$summaryLabelWidth: 130px;
+$summaryLabelWithGap: calc(#{$summaryLabelWidth} + 10px);
+$summaryPadding: 56px;
+//  drawerWindowWidth (45px)     summaryPadding left (30px)              summaryPadding right(26px)
+//    |                                |                                              |
+// [----][-------------------------][-----][------------][************************][-----]
+//              |                                |                    |
+//    sidebarWithToolstripWidth (366px)    summaryLabelWithGap   summaryContentWidth
```

Comment:
```
/*
multi
line
comment
*/
```
?

Reply:
VSCode seems to do some clever stuff to confuse us if I change it to multi line comment :sweat_smile:. Is it still worth changing?
<img width="763" alt="Screenshot 2022-04-28 at 09 16 06" src="https://user-images.githubusercontent.com/3085815/165708804-7d0f5183-f5b0-4a10-9a3a-4801df65e2c2.png">